styles/productDetailsStyles.tsx

```diff
@@ -0,0 +1,62 @@
+import {StyleSheet} from 'react-native';
```

Comment:
I can say that it's almost impossible to support such styling structure.
It's going to be a headache in the nearest future.
U can create component's own style in the same file where the component is declared or use the following structure:
<img width="282" alt="Screenshot 2021-12-27 at 00 37 00" src="https://user-images.githubusercontent.com/58725560/147420621-9286578c-f5d0-4d1d-bd4e-ddf86062547f.png">